packages/mobile/src/harmony-native/components/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,100 @@
+import { useCallback, useState } from 'react'
+
+import { css } from '@emotion/native'
+import { useLinkProps } from '@react-navigation/native'
+import { Pressable, type GestureResponderEvent } from 'react-native'
+import { Gesture, GestureDetector } from 'react-native-gesture-handler'
+import Animated, {
+  interpolateColor,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming
+} from 'react-native-reanimated'
+
+import { useTheme } from '../../foundations/theme/useTheme'
+import { Text } from '../Text/Text'
+
+import type { TextLinkProps } from './types'
+
+const AnimatedText = Animated.createAnimatedComponent(Text)
+
+export const TextLink = <ParamList extends ReactNavigation.RootParamList>(
+  props: TextLinkProps<ParamList>
+) => {
+  const {
+    to,
+    action,
+    children,
+    variant = 'default',
+    // TODO: Add external link support
+    // isExternal = false,
```

Comment:
Do we want to add this in this PR, or create a ticket to do this later?

Reply:
lets add a ticket to do later, cause there are a few things to figure out i think. react-native has a special way to handle external linking.